app/models/manageiq/providers/openstack/provider.rb

```diff
@@ -1,11 +1,11 @@
-class ProviderOpenstack < Provider
+class ManageIQ::Providers::Openstack::Provider < ::Provider
```

Comment:
@blomquisg @matthewd Merging and we can continue this discussion and implement in a future PR.